ui/app/AppLayouts/Profile/panels/ProfileShowcaseAccountsPanel.qml

```diff
@@ -26,8 +25,9 @@ ProfileShowcasePanel {
             var tmpObj = Object()
             root.roleNames.forEach(role => tmpObj[role] = showcaseObj[role])
             tmpObj.showcaseVisibility = value
-            showcaseModel.append(tmpObj)
-            showcaseVisibility = Constants.ShowcaseVisibility.NoOne // reset
+            showcaseModel.insertOrUpdateItemJson(JSON.stringify(tmpObj))
+            root.showcaseEntryChanged()
+            root.updateModelsAfterChange()
```

Comment:
Why do you have two signal calls here? In theory, the UI should update by itself after the model updates itself.

Reply:
Yup, in the backend, just take care to call `dataChanged(fromIndex, toIndex, [roles]))` whenever some model role changes; see https://doc.qt.io/qt-5/qabstractitemmodel.html#dataChanged

Reply:
I am currently working on removing redundant calls and renaming the signals. In the current implementation i _can_ do away with two sinals
<img width="559" alt="Screenshot 2023-10-27 at 17 47 44" src="https://github.com/status-im/status-desktop/assets/2522130/58bec75f-d891-4a94-830a-6461552af045">